index.js

```diff
@@ -35,7 +35,7 @@ module.exports = (awesomeTypescriptOptions = {}, nextConfig = {}) => {
 
       if (dev && !isServer) {
         config.module.rules.push({
-          test: /\.tsx?$/,
+          test: /\.(ts|tsx)?$/,
```

Comment:
because I had an error with myFile.ts file. I usually use .ts file If It's not a react component. I may be wrong, if there is another way to use .ts file then let me know.

Reply:
uhm, that's strange, because `tsx?` means `ts` or `tsx` which is exactly `(ts|tsx)`. I'll check it

Reply:
Thanks! It didn't work with my code, but maybe I was doing wrong. webpack is still complicated to me...

Reply:
Ok, I've checked it, and `tsx?` is working fine on my side for `.ts` files. Feel free to raise an issue if the error persists

Reply:
I tested again by making ts file and import from another file... and It worked!  hm... I thought to make stateless functional component with .ts file should work and It didn't... It should have been .tsx because It looks like just an arrow function, but It has html and other tsx syntax.  sorry for this stupid mistake. But If It worked from by not making stupid mistake, I wouldn't even start looking it more about this next-plugins hahaha